limesurvey/limesurvey.py

```diff
@@ -132,38 +169,81 @@ def get_survey(self, data, suffix=""): # pylint: disable=unused-argument
             "access_code": self.access_code,
         }
 
-    def get_student_access_code(self, anonymous_user_id):
+    def user_in_survey(self, anonymous_user_id: str) -> bool:
         """
-        Return the access code for the current user.
+        Check if the user is already in the survey.
+        - `params` variable is a list of parameters to pass to the API call.
+            - params[0]: Survey ID
+            - params[1]: Retrieve participants starting from this index
+            - params[2]: Maximum number of participants to retrieve
+            - params[3]: Retrieve only participants with unused tokens
+            - params[4]: List with extra participant attributes to retrieve
+            - params[5]: Dictionary of conditions to filter participants
+
+        args:
+            anonymous_user_id (str): The anonymous user ID
+
         """
-        limesurvey_api_url = getattr(settings, "LIMESURVEY_INTERNAL_API", None)
-        if not limesurvey_api_url:
-            return False
+        params = [
+            self.survey_id, 0, 1, False, ["attribute_1"], {"attribute_1": anonymous_user_id},
+        ]
```

Comment:
A named tuple would be better as it's self-explanatory

Reply:
I agree. Maybe, other option could be a dictionary. What do you think? @Ian2012 
```python
params = {
    "survey_id": self.survey_id,
    "start": 0,
    "limit": 1,
    "unused": False,
    "attributes": ["attribute_1"],
    "conditions": {"attribute_1": anonymous_user_id},
}

response = self._invoke("list_participants", *params.values())
```

Reply:
I dict is a valid option too

Reply:
Done